File: Factorial.py
n = int (input("Ingrese un numero: "))
print (" ")

if n == 1:
    print ("El fatorial del numero ingresado es:",1)
else:
    facto = 1
    for i in range (1,n+1):
        facto = int(facto*i)
    print ("El factorial del numero ingresado es: ",facto)

File: Factorial.cpp
#include <iostream>
#include <math.h>

using namespace std;
int main(){
    int n = 0, fac = 1, i = 0;
    cout<<"Ingrese un numero: "; cin>>n;

    for (i = n; i >= 1; i--){
        fac *= i;
        
    }
    cout<<"El factorial del numero ingresado es: "<<fac;

    return 0;
}